pypeit/spectrographs/gemini_gmos.py

```diff
@@ -400,10 +399,10 @@ def bpm(self, filename, det, shape=None, msbias=None):
             # Apply the mask
             xbin = int(binning.split(' ')[0])
             if xbin != 2:
```

Comment:
Perhaps the code should fault rather than embed.

Reply:
modified.